### ISSUE TITLE
Bundle font

### ISSUE THREAD

**Issue by DamonGant** [crmarsh/discord-linux-bugs#147]
Some glyphs seem to be missing from the font Discord uses, especially from some nicknames. For instance, I have a friend nicknamed to "this.is.🍙" which appears as "this.is." on Linux and normally on Windows.

These doesn't seem to be an issue in Firefox on the same machine.

**Comment by greenbigfrog** [comment]
font fallbacks are decided by the displaying browser/client/whatever (some characters aren't included in discords default font)... Some names render diffrently in chrome(ium) and FF, etc.

so either don't bother with it, or make sure you got the correct fonts installed locally

**Comment by crmarsh** [comment]
Yeah, the font we distribute doesn't have very many glyphs, we just fall back to system fonts. I'm not sure why Firefox's behavior is different in this case, I would have assumed all the browsers handle that stuff in the same way.

**Comment by crmarsh** [comment]
Whatever the case, I don't expect bundling a font with every possible glyph is the correct solution; that would be a fairly substantial waste for 99% of our users.

**Comment by DamonGant** [comment]
Firefox bundles [EmojiOne](http://emojione.com/) and falls back to system for any other glyphs. You can get them working for other browsers on Linux, including Discord, but it requires a little bit of a hacky workaround dealing with fontconfig and downloading a font. It'd just be nice to have the most commonly used ones consistent across platforms.

**Comment by schittler** [comment]
So, I want to weigh in a little more on this issue. There currently is a running gag involving the emoji :diamond_shape_with_a_dot_inside: (don't ask) on a Discord I frequent. 

I didn't really get it at first.

Then I noticed this emoji is [very inconsistent across operating systems and fonts](http://emojipedia.org/diamond-shape-with-a-dot-inside/). It's meaning is currently split across platforms and that is actually working *against* what the standard was meant to do.

This is not the only glyph that is massively different on different platforms.

Plus, this might be an extra bug, but I've noticed that Discord uses Color emoji in names for voice channel users on Windows and solid shapes on Linux, getting me even *more* confused about things.

I know this is complaining on a really high level, but please, *visual consistency matters*.
